### PR TITLE
Added namespacing for global stats

### PR DESCRIPTION
**Pull request by lchristopherson**

### CODE REVIEW

File: src/main/scala/com/twitter/parrot/server/FinagleTransport.scala [hunk 1]
```diff
@@ -33,8 +35,8 @@ import org.jboss.netty.util.CharsetUtil.UTF_8
 import com.twitter.util._
 
 object FinagleTransportFactory extends ParrotTransportFactory[ParrotRequest, HttpResponse] {
-  def apply(config: ParrotServerConfig[ParrotRequest, HttpResponse]) = {
-    val statsReceiver = new OstrichStatsReceiver
+  def apply(config: ParrotServerConfig[ParrotRequest, HttpResponse], statsName: String = "") = {
```
Comment (maniksurtani) [inline src/main/scala/com/twitter/parrot/server/FinagleTransport.scala]: Shouldn't the default be `global`?

File: src/main/scala/com/twitter/parrot/server/RequestQueue.scala [hunk 2]
```diff
@@ -29,6 +29,7 @@ class RequestQueue[Req <: ParrotRequest, Rep](
 ) {
 
   private[this] val log = Logger.get(getClass)
+  private[this] var statsName = ""
```
Comment (maniksurtani) [inline src/main/scala/com/twitter/parrot/server/RequestQueue.scala]: Shouldn't this default be `global`?

File: src/main/scala/com/twitter/parrot/server/ThriftTransport.scala [hunk 3]
```diff
@@ -30,14 +31,14 @@ import com.twitter.util.Await
 import org.apache.thrift.protocol.TBinaryProtocol
 
 object ThriftTransportFactory extends ParrotTransportFactory[ParrotRequest, Array[Byte]] {
-  def apply(config: ParrotServerConfig[ParrotRequest, Array[Byte]]) = {
+  def apply(config: ParrotServerConfig[ParrotRequest, Array[Byte]], statsName: String = "") = {
```
Comment (maniksurtani) [inline src/main/scala/com/twitter/parrot/server/ThriftTransport.scala]: `global` as default...

File: src/main/scala/com/twitter/parrot/server/FinagleTransport.scala (hunk 1 above)
Comment (lchristopherson) [inline src/main/scala/com/twitter/parrot/server/FinagleTransport.scala]: It seems that the only place that `"global"` is used is in ParrotServer.scala. Everywhere else uses `Stats` which is mapped to `Stats.get("")`. I'm not sure if this was the intended behavior but I was just trying to keep the default behavior the same.

Comment (maniksurtani) [inline src/main/scala/com/twitter/parrot/server/FinagleTransport.scala]: ok.

**Pull request merged**